src/app/search/search-notebook-table/search-notebook-table.component.scss

```diff
@@ -0,0 +1,11 @@
+.mat-column-descriptorType {
+  max-width: 90px;
+}
+
+.mat-column-projectLinks {
+  max-width: 80px;
+}
```

Comment:
Should be in `rem`. I'm wondering if these changes should be in https://github.com/dockstore/dockstore-ui2/blob/release/1.14/src/app/shared/styles/entry-table.scss (showing the file from the release branch because it's in rem) unless there's a reason why the notebooks table needs this override and not the other tables

Reply:
See above.